draft-ietf-scim-device-model.mkd

```diff
@@ -333,36 +352,30 @@ clientToken
 This attribute type string contains a token that the client will use 
 to authenticate itself. Each token may be a string up to 500 
 characters in length. It is mutable, required, case sensitive 
-and returned by default if it exists.
+and returned by default if it exists.  The SCIM server should expect
+that client tokens will be shared by the SCIM client with other
+components within the client's infrastructure.
 
-## Complex Attribute
+## Complex Attributes
 
-certificateInfo
+### certificateInfo
 
-It is the complex attribute that Contains x509 certificate's subject
-name and root CA information associated with the device control or 
-telemetry app. It further has three attributes that are described 
-below.
+It is the complex attribute that contains x509 certificate's subject
+name and root CA information associated with application clients that
+will connect for purposes of device control or telemetry.
 
-rootCN
+rootPublicKey
 
-It is the root certificate common name. This attribute is required,
-read only, singular and case sensitive.
+this is the base64 encoding of the public key of a certficate that is
+used to sign certificates that may used for client application access.
 
 subjectName
 
-Also known as the Common Name (CN), the Subject Name is a field in the
-X.509 certificate that identifies the primary domain or IP address for
-which the certificate is issued. This attribute is not required,
-read only, singular and case sensitive.
-
-subjectAlternativeName
-
-This attribute allows for the inclusion of multiple domain names and
-IP addresses in a single certificate. This enables the certificate to
-be used for multiple related domains or IPs without the need for 
-separate certificates for each. This attribute is not required,
-read only, multivalued and case sensitive.
+If present, this field contains a dnsName, as specified in Section
```

Comment:
`iPAddress` should also be valid here.

Reply:
I think we need to talk about that.

Reply:
The issue here is that it is not safe to bind this name to the transport because of NATs.  Therefore what we really have is a username.